README.md

```diff
@@ -111,6 +111,15 @@ git push -u origin <your_branch_name>
 
 **15** Voila :exclamation: You have made a PR to the awesome-javascript-projects :boom: . Wait for your submission to be accepted and your PR to be merged.
 
+### This Project is a part of the following Open Source Program
+
+<p align="center">
+<img align="center" alt="GIF" src="https://github.com/Ayush7614/Awesome-JavaScript-Projects/blob/main/cwoc.png" width="200" height="200" />
```

Comment:
Don't Use absolute path. First of all place images at right place means in images folder of assets folder. Then use` ./assets/images/cwoc.png` . Reason is very simple you are using source of image from your forked repo if you delete your forked repo then it will not be shown on this repo readme. 

Add margin to both images as well.